cnvkit_heatmap/cnvkit_heatmap.xml

```diff
@@ -0,0 +1,115 @@
+<tool id="cnvkit_heatmap" name="CNVkit Heatmap" version="@VERSION@.0" 
+      profile="17.09">
+    <description>
+        CNVKit, a command-line toolkit for copy number analysis.
+    </description>
+    
+
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+
+
+    <requirements>
+        <requirement type="package" version="0.9.4a0">cnvkit</requirement>
+    </requirements>
+
+
+    <command detect_errors="exit_code"><![CDATA[
+        @CMD_BEGIN@ heatmap 
+
+        #if $by_bin:
+        --by-bin
+        #end if
+
+        #if $restrict_analysis.restrict_to == "chromosome":
+        --chromosome ${restrict_analysis.chromosome} 
+        #else if $restrict_analysis.restrict_to == "region":
+        --chromosome ${restrict_analysis.chromosome}:${restrict_analysis.reg_start}-${restrict_analysis.reg_end} 
+        #end if
+
+        #if $desaturate:
+        --desaturate
+        #end if
+
+        #if $sample_sex == "Male":
+        --sample-sex Male
+        #else if $sample_sex == "Female":
+        --sample-sex Female
+        #end if
+
+        #if $reference_male:
+        --male-reference
+        #end if
+
+        #if $disable_xy:
+        --no-shift-xy
+        #end if
+
+        #for $sample in $samples:
+        '${sample}'
+        #end for
+
+        -o output.pdf
+    ]]></command>
+
+
+    <inputs>
+        <param name="samples" type="data" format="text,tabular,tsv" 
+               multiple="true" label="Copy Number Files"/>
```

Comment:
Should this tool allow multiple samples? In current state, the outputs would overwrite each other in the job working dir. Either need to only allow a single sample or add handling of multiple outputs.

Reply:
My understanding of the heatmap tool in CNVkit was that it could take multiple sample inputs and produce a single heatmap, where each sample was given a row in the graphic. If that's incorrect, I'll just have the tool account for a single input.